Mifare Classic Tool/app/src/main/res/values-it/strings.xml

```diff
@@ -0,0 +1,400 @@
+    <string name="text_data">Dati</string>
+    <string name="text_value">Valore</string>
+    <string name="text_from">Da</string>
+    <string name="text_to">A</string>
+    <string name="text_key_a">Chiave A</string>
+    <string name="text_key_b">Chiave B</string>
+    <string name="text_key_ab">Chiave A|B</string>
+    <string name="text_never">Mai</string>
+    <string name="text_keys_not_known">Chiavi (A e B) sconosciute</string>
+    <string name="text_block_read_only">Blocco di sola lettura</string>
+    <string name="text_atqa">ATQA</string>
+    <string name="text_sak">SAK</string>
+    <string name="text_ats">ATS</string>
+    <string name="text_no_mfc_support_device">Questo dispositivo non supporta le MIFARE Classic!</string>
+    <string name="text_no_mfc_support_tag">Non è un tag MIFARE Classic!</string>
+    <string name="text_no_tag">Non è presente nessun tag MIFARE Classic!</string>
+    <string name="text_generic_info">Informazioni generali</string>
+    <string name="text_mf_info">Informazioni su MIFARE Classic</string>
+    <string name="text_rf_tech">Tecnologia RF</string>
+    <string name="text_rf_tech_14a">ISO/IEC1443, Tipo A</string>
+    <string name="text_tag_type_and_manuf">Produttore e tipo di tag</string>
+    <string name="text_mem_size">Dimensione della memoria</string>
+    <string name="text_block_size">Dimensione del blocco</string>
+    <string name="text_sector_count">Numero di settori</string>
+    <string name="text_block_count">Numero di blocchi</string>
+    <string name="text_vb">VB</string>
+    <string name="text_vb_as_int">VB come intero</string>
+    <string name="text_copy">Copia</string>
+    <string name="text_paste">Incolla</string>
+    <string name="text_static_ac_default">08778F</string>
+    <string name="text_wait_read_tag">Leggendo il tag&#8230; (Non rimuoverlo)</string>
+    <string name="text_choose_key_files">Scegli alcuni file con le chiavi:</string>
+    <string name="title_activity_import_export_tool">Strumento Importa/Esporta</string>
+    <string name="title_activity_diff_tool">Strumento per la differenza</string>
+    <string name="text_valueblock">ValueBlock</string>
+    <string name="text_chooser_info_text">Scegli un file:</string>
+    <string name="text_no_files_in_chooser">Nessun file trovato</string>
+    <string name="text_open_dump_title">Apri dump di un tag</string>
+    <string name="text_ac_pos">Posizione:</string>
+    <string name="text_vb_location">Posizione:</string>
+    <string name="text_strange_error">È avvenuto un errore sconosciuto :(</string>
+    <string name="text_invalid_ac_or_sector_dead">Condizioni di accesso non valide o settore morto :(</string>
+    <string name="text_keys_read_only">Le chiavi (A e B) sono di sola lettura</string>
+    <string name="text_ac_read_only">Le condizioni di accesso sono di sola lettura</string>
+    <string name="text_write_key_b_not_known">Chiave col permesso di scrivere (B)         sconosciuta</string>
```

Comment:
Why are there so much spaces?

Reply:
Sorry, android studio view \n as spaces, I missed this line in the review.